tools/shaderc/shaderc.cpp

```diff
@@ -1215,7 +1248,7 @@ namespace bgfx
 			{
 				preprocessor.setDefine(glslDefine);
 			}
-			char temp[256];
```

Comment:
Does it really matter?

Reply:
Following the Boy Scout Rule in coding, which suggests always leaving the code a bit better than you found it, this change aims to elevate the quality of the line from good to excellent.

Considering the readers and future maintainers, aligning the existing macOS code path with the updated iOS one could be quite beneficial. It might seem a bit perplexing to readers why there's a divergence between them. Therefore, this code adjustment could potentially ease any cognitive load for those working with the code in the future.

Reply:
> Following the Boy Scout Rule in coding, which suggests always leaving the code a bit better than you found it, this change aims to elevate the quality of the line from good to excellent.

The problem with that philosophy is that PR then becomes full of non-related things.

What I would prefer, is:

1. focused fix/feature PR that doesn't touch things that are not relevant to PR
2. and then cleanup PR that does improve code

Actually order could be also 2, 1, depending on things you want to cleanup.

This way you're not overwhelming reviewer with large PR. I mean this would be insta-accepted if PR was simpler. 🙂

Reply:
Thank you for sharing your preference. It's so lovely that you are totally up for the Boy Scout Rule and understand its critical value. I'll happily segregate the feature work from the clean up work.